src/components/new-request/search.vue

```diff
@@ -138,10 +136,27 @@
 
       <!-- Restoration / Reinstatement flow -->
       <template v-else-if="isRestoration">
-        <EntityType cols="12" md="6" />
         <BusinessLookupFetch />
-        <Jurisdiction v-if="isSelectedXproAndRestorable" cols="12" md="4" />
-        <CompanyType v-if="showCompanyTypeRadioButtons" />
+          <CompanyType v-if="getSearchBusiness && isBcRestorable && isSupportedRestoration(getEntityTypeCd)" />
```

Comment:
The `isSupportedRestoration(getEntityTypeCd)` was added here to tackle edge cases which are for COOPs (CP) and Credit Union (FI).

For these two, we only want to show the name input and that's it.
![cp and credit union](https://github.com/bcgov/namerequest/assets/122301442/81e6631a-1636-4e62-96b3-612fe4143ad1)